compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -5848,5 +5857,6 @@ THRIFT_REGISTER_GENERATOR(
     "                     undated: suppress the date at @Generated annotations\n"
     "                     suppress: suppress @Generated annotations entirely\n"
     "    unsafe_binaries: Do not copy ByteBuffers in constructors, getters, and setters.\n"
+    "    javax_annotations: generate javax annotations (jakarta by default)\n"
```

Comment:
isn't javax the default as the status quo?

Reply:
@Jimexist You are right ! I made modification in the code.

Reply:
@Jimexist can you help me to have an account to Jira ? I sent mail twice but without response.